spec/end_to_end/end_to_end_spec.rb

```diff
@@ -58,29 +58,31 @@
     end
   end
 
-  context "when the form has a file upload" do
-    let(:file_question_text) { "Upload a file" }
-    let(:test_file) { "/tmp/temp-file.txt" }
+  unless ENV.fetch('SKIP_FILE_UPLOAD', false)
```

Comment:
Rather than setting the default here, would it make sense to add it to the default environment variables in the Rakefile?

Reply:
Oh, it is in the Rakefile (silly mistake on my part - ignore me)